lib/pf/Switch.pm

```diff
@@ -656,6 +679,9 @@ sub _parentRoleForVlan {
 sub getAccessListByName {
     my ($self, $access_list_name, $mac) = @_;
     my $logger = $self->logger;
+
+    return $self->{'_access_lists'}->{$access_list_name} if (defined($self->{'_access_lists'}->{$access_list_name}));
```

Comment:
Add a logger debug saying something like "Using ACL from the switch entry instead of the one defined in the role"